packages/core/src/modules/ledger/LedgerModule.ts

```diff
@@ -72,4 +72,12 @@ export class LedgerModule {
   public async getCredentialDefinition(id: string) {
     return this.ledgerService.getCredentialDefinition(id)
   }
+
+  public async getRevocationRegistryDefinition(revocRegDefId: string) {
+    return this.ledgerService.getRevocationRegistryDefinition(revocRegDefId)
+  }
+
+  public async getRevocationRegistryDelta(revRegId: string, from = 0, to = new Date().getTime()) {
```

Comment:
Also nit: Having the time notation in the params names (and also making it an object can make more clear what the API expects.

E.g. if it needs milliseconds: 

```suggestion
  public async getRevocationRegistryDelta({ revocationRegistryId, fromMs, toMs = 0 }: {
  	revocationRegistryId: string;
  	fromMs: number,
  	toMs?: number
  }) {
```

Reply:
I've updated it to `fromSeconds` as `fromS` looked a little peculiar to me.